lib/minidoc.rb

```diff
@@ -234,7 +234,7 @@ def create
   end
 
   def update
-    self.class.collection.update({ _id: id }, attributes)
+    self.class.collection.update({ _id: id }, { "$set" => attributes.except!(:_id) })
```

Comment:
WDYT about using `except` instead of `except!` to avoid mutation? I'm not sure if it will have side-effects but I worry it might, and it doesn't seem necessary.